### PR TITLE
{compiler} [GCC/4.9.3-2.25] Clang 3.8.0 (REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**
Needs hpcugent/easybuild-easyblocks#898

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node2611.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/700c8a88af9bb2d28f191029718e1bbc for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 10 out of 10 (7 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/9ed93d28fce0875816774a6aad36b28f for a full test report.

**Comment by wpoely86** [comment]
@boegel please review

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 6 out of 7 (7 easyconfigs in this PR)
node2611.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/633dd24aa4ddc71cb9e5ed34b48140b3 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 6 out of 7 (7 easyconfigs in this PR)
node2611.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/eff575b2de97ac476b8af742573cd7d2 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2611.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a40b4e94fe8d33280a78d4f442757fbc for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/6d579e975fed722a64dfca055a063d98 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 13 out of 13 (7 easyconfigs in this PR)
node2409.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0733b65065fe6821518bf8430d25ad5c for a full test report.

File: easybuild/easyconfigs/n/ncurses/ncurses-6.0-GCC-4.9.3-2.25.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+name = 'ncurses'
+version = '6.0'
+
+homepage = 'http://www.gnu.org/software/ncurses/'
+description = """The Ncurses (new curses) library is a free software emulation of curses in System V Release 4.0,
+ and more. It uses Terminfo format, supports pads and color and multiple highlights and forms characters and
+ function-key mapping, and has all the other SYSV-curses enhancements over BSD Curses."""
+
+toolchain = {'name': 'GCC', 'version': '4.9.3-2.25'}
```
Comment (boegel) [inline easybuild/easyconfigs/n/ncurses/ncurses-6.0-GCC-4.9.3-2.25.eb]: @wpoely86 why include this easyconfig, you're not using it?

Comment (wpoely86) [inline easybuild/easyconfigs/n/ncurses/ncurses-6.0-GCC-4.9.3-2.25.eb]: a well, a lingering easyconfig probably. Doesn't hurt to have it.

Comment (boegel) [inline easybuild/easyconfigs/n/ncurses/ncurses-6.0-GCC-4.9.3-2.25.eb]: well, no, but in some of the easyconfigs here you're hardcoding the toolchain for ncurses to `GCCcore`; it's better not to, and use this one instead?
Same for libreadline btw...

Comment (wpoely86) [inline easybuild/easyconfigs/n/ncurses/ncurses-6.0-GCC-4.9.3-2.25.eb]: You mean the other way round: I'm gonna drop this one. libreadline and ncurses should be `GCCcore`

Comment (boegel) [inline easybuild/easyconfigs/n/ncurses/ncurses-6.0-GCC-4.9.3-2.25.eb]: OK, works for me :)

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**